Requirements.txt

```diff
@@ -0,0 +1,32 @@
+Functionele eisen:
+
+    De applicatie moet verschillende gebruikersrollen ondersteunen, waaronder beheerders (Hella, Paulus, Robin) en personeelsleden, met elk hun specifieke rechten en toegangsniveaus.
```

Comment:
functionele eisen zijn voor de applicatie. Deze applicatie blijft hopelijk nog lang in gebruik. Ook nadat de hella misschien het bedrijf verlaat. Hou het bij de rollen. dat zie je namelijk ook terug in je andere eisen. In plaats van de rol te gebruiken benoem je nu een persoon.